.rubocop.yml

```diff
@@ -17,6 +18,8 @@ AllCops:
 Style/ModuleFunction:
   Enabled: false
 
+Metrics/LineLength:
```

Comment:
I think we should follow default value here

Reply:
Default and Standard - it`s good, but we are live in 2020 year, and we have 4k monitors. 80 symbols - reduce one-liners, variables, readability. Why suffer? 
I think, 120 symbols is compromise between 80 symbols and cancellations. :)

Reply:
I don't like all rubocop defaults as well (like argument alignment or method naming) but have decided to follow default rubocop as close as possible with this project.

I've changed all code generated by `rails new` to fit default config as well https://github.com/evrone/evrone_opensource/pull/2.

Reply:
ok